base/data/GreenTag.js

```diff
@@ -125,6 +125,8 @@ class GreenTag extends DataClass {
 	tag;
 	/** The size (in bytes) of the creative this tag represents */
 	weight;
+    /** For holding notes info. TODO: maybe move somewhere else? */
+    notes;
```

Comment:
Hello @dodo721 

Have you added a `String notes` field to GreenTag.java?
The server needs that to reliably remember the info.

Thanks,
Dan

Reply:
Hi Dan,
I have done so in the adserver branch of the same name! It's a class with a few fields attached.
Adserver still doesn't let me create PRs in github, I'll forward the email I sent earlier containing the details!
Thanks,
Vera